src/main_options.cpp

```diff
@@ -166,8 +166,10 @@ bool PrintSystemInformation()
     SDL_version sdl_linked_ver;
     SDL_GetVersion(&sdl_linked_ver);
 
-    printf("SDL version (compiled):  %d.%d.%d\n", SDL_MAJOR_VERSION, SDL_MINOR_VERSION, SDL_PATCHLEVEL);
-    printf("SDL version (linked):    %d.%d.%d\n", sdl_linked_ver.major, sdl_linked_ver.minor, sdl_linked_ver.patch);
+    printf("SDL version (compiled):  %hhu.%hhu.%hhu\n",
```

Comment:
N.B: Actually, all that should be replaced with C++ std::cout. Plus, this would permit to avoid the %hh c-ish stuff in c++ code. Could you do it?